index.html

```diff
@@ -686,6 +686,51 @@ <h4>Redirected and Proxied External Content</h4>
           </p>
         </section>
       </section>
+
+      <section id="state-tokens">
+        <h3>State Tokens</h3>
+        <blockquote class="informative">
+          Non-normative note: State tokens are similar to ETags, providing a mechanism for clients to track changes
+          in resource state without having to compare the entire entity body, and to perform conditional updates.
+          Like ETags, the principle behind state tokens is that implementations have the best information on
+          resource state, and can choose the most accurate and efficient validation mechanism. State tokens differ
+          from ETags in that there is no expectation the state token must change if the resource is not
+          byte-for-byte identical, when the underlying state of the resource has not changed. For example, changes
+          in the order of RDF triples, blank node identifiers, and other serialization variations would not require
+          the X-State-Token value to change.
+        </blockquote>
+        <blockquote class="informative">
+          Non-normative note: A state token can be more reliable for validation than an ETag when an implementation
```

Comment:
"validation" again... perhaps "A state token can provide reliable indication of an unchanged resource in cases where implementations could not provide strong ETags."